docs/_interface/mining.md

```diff
@@ -4,55 +4,72 @@ sort_key: F
+started, otherwise the miner will build on an outdated side chain, meaning block rewards 
+will not be recognized by the main network. 
 
 
 ## GPU Mining
 
 ### Installing Ethminer
 
-The Ethminer software can be installed from a downloaded binary or built from source. The relevant downloads 
-and installation instructions are available from the [Ethminer Github](https://github.com/ethereum-mining/ethminer/#build). Standalone executables are available for Linux, macOS and Windows.
+The Ethminer software can be installed from a downloaded binary or built from 
+source. The relevant downloads and installation instructions are available from the 
+[Ethminer Github](https://github.com/ethereum-mining/ethminer/#build). 
+Standalone executables are available for Linux, macOS and Windows.
 
 ### Using Ethminer with Geth
 
 
-An account to receive block rewards must first be defined. The address of the account is all that is required to start mining - the mining rewards will be credited to that address. This can be an existing address or one that is newly created by Geth. More detailed instructions on creating and importing accounts are available on the [Account Management](/docs/interface/managing-your-accounts) page.
+An account to receive block rewards must first be defined. The address of the account is all 
+that is required to start mining - the mining rewards will be credited to that address. This can 
+be an existing address or one that is newly created by Geth. More detailed instructions on creating 
+and importing accounts are available on the [Account Management](/docs/interface/managing-your-accounts) 
+page.
 
-The account address can be provided to `--mining.etherbase` when Geth is started. This instructs Geth to direct any block rewards to this address. Once started, Geth will sync the blockchain. If Geth has not connected to this network before, or if the data directory has been deleted, this can take several days. Also, enable HTTP traffic with the `--http` command.
+The account address can be provided to `--mining.etherbase` when Geth is started. This instructs Geth 
+to direct any block rewards to this address. Once started, Geth will sync the blockchain. If Geth 
+has not connected to this network before, or if the data directory has been deleted, this can take 
+several days. Also, enable HTTP traffic with the `--http` command.
 
 ```shell
 geth --http --miner.etherbase 0xC95767AC46EA2A9162F0734651d6cF17e5BfcF10
 ```
 
-The progress of the blockchain syncing can be monitored by attaching a JavaScript console in another terminal. More detailed information about the console can be found on the [Javascript Console](/docs/interface/javascript-console) page. To attach and open a console:
+The progress of the blockchain syncing can be monitored by attaching a JavaScript console in another 
+terminal. More detailed information about the console can be found on the 
```

Comment:
A lot of "changes" here are because you have changed the linebreaks/whitespace. Makes it more difficult to asses the differences. In the future, please try to do either content-changes XOR whitespace/linebreak changes in the same PR.